Well-Coffee-UI/src/App.jsx

```diff
@@ -1,18 +1,111 @@
 import NavigationBar from "./components/navigation/NavigationBar";
-import NavigationRoutes from "./components/navigation/NavigationRoutes";
-import Footer from "./components/footer/Footer";
+import { useState } from "react";
+import { Route, 
+  Routes, 
+  Navigate } from "react-router-dom";
+import HomePage from "./components/pages/HomePage";
+import AccountPage from "./components/pages/AccountPage";
+import OrderPage from "./components/pages/OrderPage";
+import ManageEmployees from "./components/pages/ManageEmployees";
+import DailyReport from "./components/pages/DailyReport";
+import Tutorial from "./components/pages/Tutorial";
+import LoginPage from "./components/pages/LoginPage";
+import Register from "./components/pages/Register";
+import LogoutPage from "./components/pages/LogoutPage";
+
+
 
 function App() {
 
+  const [authenticated, setAuthenticated] = useState(false);
+
   return (
-    <div className="d-flex flex-column min-vh-100">
-      <NavigationBar />
-      <main className="flex-grow-1">
-        <NavigationRoutes />
-      </main>
-      <Footer />
-    </div>
+    <>
+    <NavigationBar authenticated={authenticated} />
+    <Routes>
+      {!authenticated ? (
+        <>
+        <Route path="/login" element={<LoginPage setAuthenticated={setAuthenticated} />} />
+        <Route path="/register" element={<Register />} />
+        <Route path="*" element={<Navigate to="/login" />} />
+        </>
+      ) : (
+        <>
+          <Route path="/home" element={<HomePage />} />
+          <Route path="/order" element={<OrderPage />} />
```

Comment:
This needs to be changed to 'element={<OrderFormPage />}' in order to render the correct component, didn't realize the one in pages wasn't being used!